### DISCUSSION TITLE
Radon

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#75] [Planning consideration]
## Radon

Radon is a naturally occurring radioactive gas which may be found in high concentrations in indoor environments.

Need: Understanding the presence of radon is essential for homeowners, landowners and developers, to ensure that sufficient mitigation is taken to prevent any harmful effects.

**Comment by colmjude** [comment]
### Existing national dataset

There is an existing national dataset published by the British Geological Survey

Documentation page for [Radon data: indicative atlas for Great Britain – ESRI shapefile](https://www.bgs.ac.uk/download/radon-potential-indicative-atlas-data-for-great-britain/)

Not sure what the endpoint is because the link behind the button is the same link as the page which suggests it is a JS link

**Reply by colmjude** [reply]
You can download an[ ERSI Shapefile](https://www.bgs.ac.uk/download/radon-potential-indicative-atlas-data-for-great-britain/)

We need to get a sample of the data as GeoJSON

**Comment by DJG-DS** [comment]
[radon_dataset_analysis_report.docx](https://github.com/user-attachments/files/17375093/radon_dataset_analysis_report.docx)

**Reply by DJG-DS** [reply]
@colmjude

**Reply by colmjude** [reply]
Thanks @DJG-DS 

## What we found

### Folder contents:

The following files were found in the folder:

- Radon_Colours.xlsx: Size: 9631 bytes
- Radon_Indicative_Atlas_v3.CPG: Size: 5 bytes
- Radon_Indicative_Atlas_v3.dbf: Size: 13812712 bytes
- Radon_Indicative_Atlas_v3.lyr: Size: 8192 bytes
- Radon_Indicative_Atlas_v3.prj: Size: 417 bytes
- Radon_Indicative_Atlas_v3.sbn: Size: 2296492 bytes
- Radon_Indicative_Atlas_v3.sbx: Size: 96068 bytes
- Radon_Indicative_Atlas_v3.shp: Size: 32956436 bytes
- Radon_Indicative_Atlas_v3.shp.xml: Size: 15429 bytes
- Radon_Indicative_Atlas_v3.shx: Size: 1938708 bytes
- User_Guide_Radon_Potential_GB_v3.pdf: Size: 2516837 bytes

Total Folder Size: 53650927 bytes

### Shapefile analysis

Geometry Type: Polygon
Number of Records: 242326
Features/Attributes: TILE, CLASS_MAX, VERSION, geometry
Coordinate Reference System (CRS): EPSG:27700
File Size: 32956436 bytes


### Analysis of `CLASS_MAX` field

Min value: 1
Max value: 6
Median: 2.0
Unique Values: [1, 2, 3, 4, 5, 6]

### Plots

There are records for every 1km square in UK. Each record has a `CLASS_MAX` field which indicates the radon risk. We can plot these to show the distribution.

![radon-plot](https://github.com/user-attachments/assets/7028b1f3-c599-4b70-9b77-3ff3b5b65274)

### Sample data

Sample GeoJSON record
```
{ "type": "Feature",
  "properties": { "TILE": "NL5479", "CLASS_MAX": 1, "VERSION": "Radon Indicative Atlas GB v3" }, 
  "geometry": { "type": "Polygon", "coordinates": [ [ [ 54000.0, 779000.0 ], [ 54000.0, 780000.0 ], [ 55000.0, 780000.0 ], [ 55000.0, 779000.0 ], [ 54000.0, 779000.0 ] ] ] }
}
```

Properties (as a table)

| field | value |
| --- | --- |
| TILE | NL5479 |
| CLASS_MAX | 1 |
| VERSION | Radon Indicative Atlas GB v3 |
| geometry | [ [ [ 54000.0, 779000.0 ], [ 54000.0, 780000.0 ], [ 55000.0, 780000.0 ], [ 55000.0, 779000.0 ], [ 54000.0, 779000.0 ] ] ] |

**Comment by drewhardy77** [comment]
Background on the use of radon data in planning and the indicative atlas.

1. Radon is a mitigable development risk, and Building Regulations in England and Wales require the following based on predicted radon levels:

| Radon level (% of properties predicted to be above the Action Level*)| Mitigation required in new builds| 
|--------|--------|
| <3% | None| 
| 3-10%| Basic|
| >10%| Full| 

_*Action Level = 200Bq/m3_

3. Whilst the indicative atlas gives a good indication of areas where protection is not required, it is not granular enough to be used in areas where the risk is higher. UK planning authorities have free access to [more detailed UKHSA/BGS data on radon](https://www.bgs.ac.uk/datasets/radon-data-radon-potential-dataset/) at a 50m (with 25m buffer) scale which can and should be used for applying radon protection conditions to developments. This data is also commercially available so cannot be put on the platform as open data. In practice many authorities rely solely on the indicative atlas, but this is a sub-optimal outcome and can often lead to excessive mitigation being required.
<img width="319" alt="radon_comparison" src="https://github.com/user-attachments/assets/40952216-dfe5-4719-9f6f-63340d119379" />

4. Whilst both the radon potential datasets indicate the potential radon levels in new builds, actual radon levels in existing dwellings can be measured using detection kits over a 3 month period. This is the only accurate way of determining actual radon levels in a property as opposed to the likelihood of radon being present above the Action Level.
5. Radon mitigation is usually very effective and can add c.£1,000-£3,000 to the construction costs of a new home, depending on the size of the dwelling and the level of protection required.
